gulpfile.js

```diff
@@ -62,12 +62,24 @@ gulp.task('tslint', function () {
 });
 
 function test() {
+    // Defaults
+    var pattern = "extensionContext";
+    var invert = true;
+
+    // Check if arguments were passed
+    var patternIndex = process.argv.indexOf("--pattern");
```

Comment:
I think this runs on node 0.12 on the build server, so I don't think we can uses classes just yet...
I'll extract it to a function though...